lib/dsl/validator.js

```diff
@@ -67,6 +67,11 @@ const configPropsValidations = {
     pattern: ALPHANUMERIC_DASH,
     msg: 'blueprint property'
   },
+  BLUEPRINTS: {
+    type: 'list',
+    pattern: ALPHANUMERIC_DASH,
```

Comment:
Can be lower too.

Reply:
Here's the regex: `/^[A-Za-z][A-Za-z0-9-]*$/`, it takes the lowercase letters into account

Reply:
Ops I mean 
```suggestion
    pattern: ALPHANUMERIC_DASH_LOWER,
```

Reply:
Ah! you're talking about renaming it! Ah yes, it's a good idea.

Reply:
ALPHANUMERIC_DASH_LOWER don’t exists now.

Reply:
I thought you meant that we should rename the variable as it's misleading and doesn't tell it also supports the lowercase letters when it actually does.

Reply:
alpha numeric could also mean "the 'common' alphabet including uppercase and lowercase letters + numbers" though

Reply:
You can use this one:
`((@[a-z0-9-~][a-z0-9-._~]*)\/)?([a-z0-9-~][a-z0-9-._~]*)`
It can be a valid npm package name

Reply:
NPM package names are a mess.
https://docs.npmjs.com/files/package.json#name states some rules, but not all of them... There's also [this package](https://www.npmjs.com/package/validate-npm-package-name) from NPM (that seems to be abandoned).

Reply:
It seems the package is the officially used one at NPM, so its rules are still used... I'm making a regex out of it for blueprints

Reply:
You can take a look at https://github.com/dword-design/package-name-regex

Reply:
Yeah but it's wrong. `/^(@[a-z0-9-~][a-z0-9-._~]*\/)?[a-z0-9-~][a-z0-9-._~]*$/.test('aaa~')` returns true. Do an NPM init with the same test string and see it fail.

Reply:
I've opened an issue in the repo BTW.